pkg/actions/addon/addon.go

```diff
@@ -36,10 +34,6 @@ type Manager struct {
 }
 
 func New(clusterConfig *api.ClusterConfig, eksAPI eksiface.EKSAPI, stackManager manager.StackManager, withOIDC bool, oidcManager *iamoidc.OpenIDConnectManager, clientSet kubeclient.Interface, timeout time.Duration) (*Manager, error) {
-	if err := supportedVersion(clusterConfig.Metadata.Version); err != nil {
```

Comment:
One concern I have with removing this validation is that there might still be existing 1.18 clusters that EKS hasn't upgraded to 1.19 yet. It takes some time for them to gradually upgrade unsupported cluster versions after removing support for creating new clusters on that version. 

A safer approach would be to keep this validation and remove it after a few weeks.

Reply:
@cPu1 addons are supported on 1.18, so I don't think there is any harm as there isn't any 1.17 clusters left

Reply:
Ah, so this validation could've been removed some time after removing support for 1.17.

Reply:
yeah!